### ISSUE TITLE
Training my own data, the loss does not drop

### ISSUE THREAD

**Issue by MOMOANNIE** [MASILab/3DUX-Net#40]
Hello, 
when I train my own data, I set `dataset == 'flare', out_classes = 2`, when the training reaches 10,000 steps, why the loss does not drop?
![image](https://github.com/MASILab/3DUX-Net/assets/26375536/6657fe83-69e6-415c-a578-3327439695c6)
The loss curve is as follows:
![image](https://github.com/MASILab/3DUX-Net/assets/26375536/ed45fd64-a107-4035-a780-e6faa966ee6f)

**Comment by leeh43** [comment]
Hi, thank you for your interest. The problem occurred here is related to the cropping. You can see there is a warning "N forground 0, background #######, unable to generate class balanced samples". Please double check your label input and the foreground region within the label. Seems like there is no foreground within the label region for them to crop.

**Comment by MOMOANNIE** [comment]
> Hi, thank you for your interest. The problem occurred here is related to the cropping. You can see there is a warning "N forground 0, background #######, unable to generate class balanced samples". Please double check your label input and the foreground region within the label. Seems like there is no foreground within the label region for them to crop.

Thank you for answering my question in time, I checked the image and label of the training data. The following two pictures are the corresponding image and label. There will be multiple foregrounds in the label image, but some foregrounds will be very small. Will this affect it?

![image](https://github.com/MASILab/3DUX-Net/assets/26375536/8c2c715a-c473-4afc-b612-b63d41a3fa6c)
![image](https://github.com/MASILab/3DUX-Net/assets/26375536/2163e273-61c0-4f72-bc15-57f297d5e05b)

**Comment by s-j-y1** [comment]
您好，想请教一下如何换成自己的数据集

**Comment by ghost** [comment]
Hello, the dataset I use is also small target binary segmentation, does this model work reasonably on your dataset?

**Comment by BennettLandman** [comment]
I am closing the older bug reports as these were missed. We are now better tracking reports across the organization. Please re-open if this continues to be a blocker.

**Comment by RY-97** [comment]
请问你，使用你的数据时，有没有生成.json文件
